### DISCUSSION TITLE
install Opensnitch from https://snapcraft.io/store

### DISCUSSION THREAD

**Discussion by danutc** [evilsocket/opensnitch#366] [Q&A]
Hi

Do you have any plans to publish Opensnitch on https://snapcraft.io/store ?

Thanks
Dan

**Comment by gustavo-iniguez-goya** [comment]
Hi @danutc ,

Not in the short term (i.e.: this week). However I've just generated a snap package (I didn't expect to generate it so easily..). I'll test it along with other things that I have on the TODO list, and if it works I'll publish it.

**Reply by danutc** [reply]
Hi @gustavo-iniguez-goya 

Thanks for the great work. I am surprised that linux desktops do not have a built in application firewall. Security is a so important issue so any desktop should have a built in application firewall.

In my opinion adoption is the main hurdle for Opensnitch. Desktop users are more and more used to install applications from application stores. To me if you publish Opensnitch in an application store it becomes visible to masses and it is likely to be more quickly adopted.

Again, tanks for the great work! you fill a gap that should be provided by core OS components.

**Reply by bademux** [reply]
@gustavo-iniguez-goya could you please share your work on openstinch snapification?
What do you think about distributing UI as snap (lot of mowing parts), while golang service will be provided as pure deb. Does it make sense?

holly war note: I'm not sure snap is the best solution, I think it is reasonable "given" on ubuntu.
